fahclient/values.yaml

```diff
@@ -70,10 +74,27 @@ securityContext:
       - ALL
   readOnlyRootFilesystem: false
 
+# Don't forget to enable external access in configmap, to access service
 service:
   type: ClusterIP
   port: 80
 
+# Ingress for WebUI
+# Don't forget to enable external access in configmap, too
+ingress:
+  enabled: false
+  hostname: chart-example.local
+  path: /
+
+  # Enable TLS for Ingress e.g. letsencrypt using cert-manager
```

Comment:
You should probably mention here that the secret name has an expected value, or add a new variable to specify it, for those not using cert-manager.